src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationNodes.vb

```diff
@@ -917,8 +917,8 @@ _operationFactory.CreateFromArray(Of BoundExpression, IOperation)(_boundForToLoo
         Private ReadOnly _operationFactory As VisualBasicOperationFactory
         Private ReadOnly _returnedValue As BoundNode
 
-        Friend Sub New(operationFactory As VisualBasicOperationFactory, returnedValue As BoundNode, kind As OperationKind, semanticModel As SemanticModel, syntax As SyntaxNode, type As ITypeSymbol, constantValue As [Optional](Of Object), isImplicit As Boolean)
-            MyBase.New(kind, semanticModel, syntax, type, constantValue, isImplicit)
+        Friend Sub New(operationFactory As VisualBasicOperationFactory, returnedValue As BoundNode, refKind As RefKind, kind As OperationKind, semanticModel As SemanticModel, syntax As SyntaxNode, type As ITypeSymbol, constantValue As [Optional](Of Object), isImplicit As Boolean)
```

Comment:
>refKind As RefKind [](start = 100, length = 18)

Is it ever not None? Consider getting rid of the parameter otherwise.

Reply:
Done in ff7f6eabaf5acd84e1b64eef4c01db273c5ac101